packages/commonwealth/client/scripts/views/pages/CommunityManagement/CommunityProfile/Form/Form.tsx

```diff
@@ -0,0 +1,411 @@
+            </div>
+
+            <div className="controls">
+              {communityTagOptions.map((option) => (
+                <CWButton
+                  key={option}
+                  type="button"
+                  label={option}
+                  buttonWidth="narrow"
+                  buttonType={
+                    selectedCommunityTags.includes(option)
+                      ? 'primary'
+                      : 'secondary'
+                  }
+                  onClick={(e) => {
+                    e.preventDefault();
+                    setSelectedCommunityTags((prevTags) =>
+                      prevTags.includes(option)
+                        ? [...prevTags].filter((tag) => tag !== option)
+                        : [...prevTags, option],
+                    );
+                  }}
+                />
+              ))}
+            </div>
+          </section>
+
+          <section className="stages-section">
+            <div className="header">
+              <CWText type="h4">Stages</CWText>
+              <CWText type="b1">
+                <p>
+                  Show proposal progress on threads
+                  <CWToggle
+                    hookToForm
+                    size="large"
+                    name="hasStagesEnabled"
+                    onChange={(e) => setIsCustomStagesEnabled(e.target.checked)}
+                  />
+                </p>
+              </CWText>
+            </div>
+
+            <CWTextInput
+              label="Custom stages"
+              placeholder="[“Stage 1”, “Stage 2”]"
+              name="customStages"
+              fullWidth
+              hookToForm
+              disabled={!isCustomStagesEnabled}
+            />
+          </section>
+
+          <section className="banner-section">
+            <div className="header">
+              <CWText type="h4">Banner</CWText>
+              <CWText type="b1">
+                Display a message across the top of your community
+              </CWText>
+            </div>
+
+            <CWTextInput
+              label="Banner"
+              placeholder="Enter text to create a banner"
+              name="communityBanner"
+              fullWidth
+              hookToForm
+            />
+          </section>
+
+          <section className="default-page-section">
+            <div className="header">
+              <CWText type="h4">Default page</CWText>
+              <CWText type="b1">
+                Select the landing page for your community
+              </CWText>
+            </div>
+
+            <div className="controls">
+              <CWRadioButton
+                label="Discussions"
+                value={DefaultPage.Discussions}
+                name="defaultPage"
+                hookToForm
+              />
+              <CWRadioButton
+                label="Overview"
+                value={DefaultPage.Overview}
+                name="defaultPage"
+                hookToForm
```

Comment:
I should be able to select radio button when clicking on label. It works like that in components page 

https://github.com/hicommonwealth/commonwealth/assets/14819225/e7ff4a7e-9d1f-4598-bb83-3c8308e6c24f

Reply:
hmm, i saw the components page and it looks like the Radio buttons there are using old designs. In new designs the labels are not clickable, but i do agree that that should be clickable, fixing them....

Reply:
Done